code/controllers/subsystem/timer.dm

```diff
@@ -21,12 +21,12 @@ var/datum/subsystem/timer/SStimer
 		can_fire = 0 //nothing to do, lets stop firing.
 		return
 	for (var/datum/timedevent/event in processing)
-		//                    hacky i know, but its the only way to ensure this works with clients
+		// hacky i know, but its the only way to ensure this works with clients
```

Comment:
you can actually remove this comment, it refers to a check i ended up not needing but forgot to remove this comment above it.